# Corrects a bogus change in mips-tulip.patch

This patch fixes the invalid MAC address problem on the Qube2700.

P.

--- build-mipsel-none-r5k-cobalt.orig/drivers/net/tulip/de2104x.c	2006-08-16 16:03:18.000000000 +0100
+++ build-mipsel-none-r5k-cobalt/drivers/net/tulip/de2104x.c	2006-08-17 17:03:34.000000000 +0100
@@ -1779,12 +1779,14 @@
 		((u16 *)ee_data)[i] =
 			le16_to_cpu(tulip_read_eeprom(de->regs, i, ee_addr_size));
 
+#ifndef CONFIG_MIPS_COBALT
 	/* DEC now has a specification but early board makers
 	   just put the address in the first EEPROM locations. */
 	/* This does  memcmp(eedata, eedata+16, 8) */
 	for (i = 0; i < 8; i ++)
 		if (ee_data[i] != ee_data[16+i])
 			sa_offset = 20;
+#endif
 
 	/* store MAC address */
 	for (i = 0; i < 6; i ++)
